pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/HLRealtimeSegmentDataManager.java

```diff
@@ -285,11 +286,13 @@ public void run() {
           _segmentLogger.info("Indexed {} raw events", _realtimeSegment.getNumDocsIndexed());
           File tempSegmentFolder = new File(_resourceTmpDir, "tmp-" + System.currentTimeMillis());
 
+          SegmentZKMetadataConfig segmentZKMetadataConfig = new SegmentZKMetadataConfig();
```

Comment:
you can skip this change for HLC

Reply:
Passing the SegmentZKMetadataConfig is still needed though.. Should I pass null instead?